### PR TITLE
PRM Automated Test - LPS-177320| master

### PR DESCRIPTION
**Pull request by MGabriellaS**
[LPS-177320](https://issues.liferay.com/browse/LPS-177320)

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 1]
```diff
@@ -1095,4 +1095,38 @@ definition {
...
+				countryName = "Global",
+				followUpStrategy = "test",
+				goalsOptions = "Lead generation",
+				leadFollowUpList = "Other",
+				leadList = "No",
+				tactic = "Other",
+				targetMarketOptions = "Education,Energy",
+				targetOfLeads = "test",
+				targetRolesOptions = "Administrator",
+				typeOfActivity = "Miscellaneous Marketing");
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:1122]: Hi @MGabriellaS, could you please change it to create the MDF via API?

File: test.properties [hunk 2]
```diff
@@ -870,6 +870,16 @@
...
+##
+## Partner Portal Salesforce
+##
+
+    site.initializer.partner.portal.login.url=https://test.salesforce.com
+    site.initializer.partner.portal.consumer.key=3MVG9.hNadsS7GK6GmwkauP3.KEFV0QK5e0ZrR1dnGRcAaDJgZJRUyX2wu3h96vp3B7PQFEKwN5QXusVAFUTz
+    site.initializer.partner.portal.consumer.secret=816A8DBCF7658F484891DC492466FE40E2AEEB488DE8CA3793D1F979F3F74E96
+    site.initializer.partner.portal.username=jair.medeiros@liferay.com.prm
+    site.initializer.partner.portal.password=U*V!q6hd7DX@FrR6Dfxo
+
```
Comment (AfonsoHSantos) [inline test.properties:882]: Hi @MGabriellaS, it shouldn't be sent with your test. It's a change that remains only locally.

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
Hi @MGabriellaS, thanks for working on it!
I've resent this test with some improvements here: #3854

**Pull request closed without merging**